conf/application.conf

```diff
@@ -129,6 +129,12 @@ microservice {
       bearer-token = token
       environment = ist0
     }
+    des {
+      host = localhost
```

Comment:
we should also update app-config-base to point this to our stub

Reply:
as in not `localhost` in app-config-base and port `443` with protocol `https` - locally in application.conf we need the port to be `10109` to point to our stub

Reply:
thanks, updated the port in `application.conf`